src/queries.js

```diff
@@ -1,16 +1,25 @@
 import { gql } from '@apollo/client';
 
 export const GET_SINGLE_ALBUM = gql`
+  query ($title: String!) {
```

Comment:
would love to know why this wasnt working

Reply:
Perhaps it was just early in the process. Thankful it works now!